aggregator/src/config.rs

```diff
@@ -94,6 +94,19 @@ impl DbConfig {
     }
 }
 
+/// Makes a best-effort attempt to redact the password from the database URL, so that it is safe
+/// to display in logs.
+fn format_database_url(url: &Url, fmt: &mut std::fmt::Formatter) -> Result<(), std::fmt::Error> {
+    match url.password() {
+        Some(_) => {
+            let mut url = url.clone();
+            url.set_password(Some("REDACTED")).ok();
```

Comment:
nit: I think this is the more idiomatic way to ignore a Result
```suggestion
            let _ = url.set_password(Some("REDACTED"));
```